lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -1654,13 +1654,20 @@ describe('GatherRunner', function() {
     });
 
     it('should parse the manifest when found', async () => {
+      // https://github.com/GoogleChrome/lighthouse/pull/10136 will allow
```

Comment:
```suggestion
      // TODO: https://github.com/GoogleChrome/lighthouse/pull/10136 will allow
```

?
otherwise it feels like we'll never find this when searching for things to fix :)

Reply:
this is merged now